### PR TITLE
feat(presets): java bazelrc options

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: .aspect/bazelrc/java.bzl [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Pin java versions
+common --java_language_version=17
+common --java_runtime_version=remotejdk_17
```
Comment (gregmagolan) [inline .aspect/bazelrc/java.bzl:3]: Also a short intro at the top of the file like we do for other: https://github.com/bazel-contrib/bazel-lib/blob/0db9fbe519b6cc2c7d0f3e66e600914b537202a0/.aspect/bazelrc/javascript.bazelrc#L1

Comment (alexeagle) [reply .aspect/bazelrc/java.bzl:3]: maybe suggested edits? I'm so underwater, won't get to this

File: .aspect/bazelrc/java.bzl [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+common --tool_java_language_version=17
+common --tool_java_runtime_version=remotejdk_17
```
Comment (gregmagolan) [inline .aspect/bazelrc/java.bzl:5]: For each flag we also give the user a hand and drop the docs snippet inline and a link to the docs for that flag.
https://github.com/bazel-contrib/bazel-lib/blob/0db9fbe519b6cc2c7d0f3e66e600914b537202a0/.aspect/bazelrc/javascript.bazelrc#L9

File: .aspect/bazelrc/java.bzl (hunk 1 above)
Comment (gregmagolan) [reply .aspect/bazelrc/java.bzl:3]: low priority. I can take it over when I have a minute.

Comment (alexeagle) [reply .aspect/bazelrc/java.bzl:3]: I got it now :)

**Review by gregmagolan — APPROVED** [review]
🌮

**Pull request merged**